src/main/scala/com/gnosly/fluentsequence/view/formatter/ActorFormatter.scala

```diff
@@ -15,7 +14,7 @@ class ActorFormatter(preRenderer: PreRenderer) {
 				return new Variable2DPoint(LEFT_MARGIN, TOP_MARGIN)
 			else {
 				return new ReferencePoint(Actor.topLeft(actor.id - 1))
-					.right(max(Reference1DPoint(ViewMatrix.column(actor.id - 1)), Fixed1DPoint(DISTANCE_BETWEEN_ACTORS)))
+					.right(Reference1DPoint(ViewMatrix.column(actor.id - 1)) max Fixed1DPoint(DISTANCE_BETWEEN_ACTORS))
```

Comment:
It looses a little in readability. But it is common in scala a method `max`. Good